ServiceWebsite/ServiceWebsite.AcceptanceTests/Constants/SelfTestPageNames.cs

```diff
@@ -3,5 +3,10 @@
     public class SelfTestPageNames
     {
         public const string CheckYourComputer = "check your computer";
+        public const string UseCameraAndMicrophone = "use camera and microphone again";
```

Comment:
These have to be renamed

Reply:
Actually, you need to merge Master into this branch and you will have my changes

Reply:
Merged master into this branch and also made the required changes